functional_plant_indicators_seminat.Rmd

```diff
@@ -2427,7 +2427,7 @@ points(res.seminat.ASO[res.seminat.ASO$fp_ind=="Nitrogen2" & res.seminat.ASO$NiN
 legend("topleft", legend=c("reference","field data"), pch=c(NA,1), lty=1, col=c("black","red"))
 
 ```
-The figure shows that deviations mainly occur in V2-C1, which represents limestone-poor swamp forests. According to the functional signature from the plant community composition many of the occurrences of this ecosystem type are too acidic and may have too low availability of nitrogen. 
+The figure shows that deviations mainly occur in V2-C1 [?], which represents limestone-poor swamp forests. According to the functional signature from the plant community composition many of the occurrences of this ecosystem type are too acidic and may have too low availability of nitrogen. 
```

Comment:
Where does V2-C1 come from? Probably not intended, as the figure is not about forests. Perhaps V10-C1?